### ISSUE TITLE
Organograma Interativo

### ISSUE THREAD

**Issue by valessiobrito** [plonegovbr/brasil.gov.portal#193]
É comum encontrar no Portal Padrão na parte "Institucional" um Organograma "feio" em flash ou imagem;

Sugiro criar um produto padrão para isso, se possível aproveitando o código desse projeto:
http://organograma.yuricamara.com.br/

A ideia é que cada item no Organograma seja um link para uma página com mais detalhes da Unidade/Coordenação.

**Comment by garotadpi** [comment]
Concordo com o "feio"para flash... mas a solução apresentada não funciona com teclado.

**Comment by valessiobrito** [comment]
Como comentei, aproveitando o código, logo, será necessário complementar / alguns ajustes para ficar compatível com e-Mag, acesssibilidade; etc.
